webservices/schemas.py

```diff
@@ -348,6 +348,7 @@ class CommitteeSearchListSchema(ApiSchema):
 FilingsSchema = make_schema(
     models.Filings,
     fields={
+        'cycle': ma.fields.Int(),
```

Comment:
Do we need to add `cycle` here? Model fields should be added to the schemas automatically--we only include `pdf_url` and `document_description` because they're property methods.